python/ray/scripts/scripts.py

```diff
@@ -595,6 +599,7 @@ def start(
 
     resources = parse_resources_json(resources, cli_logger, cf)
     labels_dict = parse_node_labels_json(labels, cli_logger, cf)
+    validate_node_labels(labels_dict, cli_logger, cf)
```

Comment:
we should validate in all places where we can specify node labels: a central place to check is probably node.py